aspnetcore/blazor/hosting-models.md

```diff
@@ -131,13 +135,13 @@ Blazor Server apps should be optimized to minimize UI latency by reducing networ
 
 ### Connection to the server
 
-Blazor Server apps require an active SignalR connection to the server. If the connection is lost, the app attempts to reconnect to the server. As long as the client's state is still in memory, the client session resumes without losing state.
+Blazor Server apps require an active SignalR connection to the server. If the connection is lost, the app attempts to reconnect to the server. As long as the client's state remains in memory, the client session resumes without losing state.
```

Comment:
I think it's worth clarifying in `which` memory the state should remain for the connection to resume without issues.

Reply:
Done on https://github.com/dotnet/AspNetCore.Docs/pull/20868/commits/2052dc0e45ac31e4866ac02adef2c79a17d68033.